src/QsCompiler/Tests.Compiler/TestCases/LinkingTests/IntrinsicResolution.qs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The one thing that is not currently giving any kind of error is if there is an intrinsic callable in the compilation that is not contained in teh environment tree, correct? 
I would be good if we could also verify this, but it may make sense to add this in a separate PR when we've had a conversation on how to integrate the existing targeting.